### ISSUE TITLE
Recent Twitter error -- data['utc_offset'] float conversion.

### ISSUE THREAD

**Issue by nullism** [bbangert/velruse#128]
Hello,

As of yesterday I started noticing the following in Apache's error.log. I am able to reproduce. There have not been any code changes recently, so I am under the assumption that something changed with Twitter's API.

```
 Traceback (most recent call last):
   File "/usr/local/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 471, in __call__
     return app(environ, start_response)
   File "/usr/local/lib/python2.7/dist-packages/pyramid/router.py", line 251, in __call__
     response = self.invoke_subrequest(request, use_tweens=True)
   File "/usr/local/lib/python2.7/dist-packages/pyramid/router.py", line 227, in invoke_subrequest
     response = handle_request(request)
   File "/usr/local/lib/python2.7/dist-packages/pyramid/tweens.py", line 21, in excview_tween
     response = handler(request)
   File "/usr/local/lib/python2.7/dist-packages/pyramid/router.py", line 117, in handle_request
     root = root_factory(request)
   File "/usr/local/lib/python2.7/dist-packages/velruse/providers/twitter.py", line 158, in callback
     offset = float(data['utc_offset']) / 3600
 TypeError: float() argument must be a string or a number
```

**Comment by nullism** [comment]
Fixed. I'm going to attempt a pull request.

**Comment by mmerickel** [comment]
Fixed by #129
